arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -863,25 +929,29 @@ ::get_configuration() const
 // Set this algorithm's properties via a config block
 void
 ffmpeg_video_input
-::set_configuration(vital::config_block_sptr in_config)
+::set_configuration( vital::config_block_sptr in_config )
 {
-  // Starting with our generated vital::config_block to ensure that assumed values are present
-  // An alternative is to check for key presence before performing a get_value() call.
+  // Starting with our generated vital::config_block to ensure that assumed
```

Comment:
we should add a period after "present" reflow this comment text. to avoid weird line breaks

Reply:
Also, since we're at it, I wonder if this should be "Start" instead of "Starting"? (BTW, this comment has been cargo-culted to a lot of places; if we change it, we might want to make note of the change and apply it elsewhere. Or maybe not change it now and change it in many places in a separate PR.)

Reply:
Maybe lets keep the "start" part of the comment the same then for consistancy until this issue can be fixed for everything.